src/cpp/command.cpp

```diff
@@ -30,6 +30,45 @@
 
 using namespace SmartRedis;
 
+Command::Command(const Command& cmd)
+{
+    this->_fields = std::vector<std::string_view>(cmd._fields);
+    this->_cmd_keys = std::unordered_map<std::string_view, size_t>(cmd._cmd_keys);
+    std::vector<char*>::const_iterator local_fields_it = cmd._local_fields.cbegin();
+    std::vector<char*>::const_iterator local_fields_it_end = cmd._local_fields.cend();
+    while(local_fields_it != local_fields_it_end) {
+        int field_size = std::strlen(*local_fields_it);
```

Comment:
Relying on ``strlen`` is problematic because fields are not guaranteed to be null-terminated or might contain null-terminating characters in the middle of the field (i.e. tensor data).  We might need to update/rearrange the Command data structures so we more easily track the length of the allocated memory along with the pointer.

Reply:
So then we need to keep track of the length of the allocated memory by storing the length whenever a field is pushed to the back of `_local_fields`. What do you think about adding a new data member that is a vector of ints to the `Command` data structure? The ith element in this vector corresponds to the length of the allocated memory of the ith element in `_local_fields`. @mellis13